deepchem/models/torch_models/chemberta.py

```diff
@@ -0,0 +1,174 @@
+"""
+Huggingface/transformers RoBERTa model for sequence-based property prediction.
+"""
+
+from numpy.lib.npyio import save
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.utils.data import random_split
+import os
+
+class ChemBERTaMaskedLM(nn.Module):
+    # defaults here match HF model: DeepChem/SmilesTokenizer_PubChem_1M.
+    # TODO - figure out which to use as default, and which to pass to model class
+    def __init__(self,
+                vocab_size =600,
+                max_position_embedddings = 515,
+                number_attention_heads = 12,
+                num_hidden_layers = 6,
+                type_vocab_size = 1,
+                dataset_path: str = '',
```

Comment:
This design requires the model to have the dataset specified in the constructor which doesn't match the rest of DeepChem's design.

Reply:
Good point! For now, I've moved the dataset initialization, data collator, + training args to the `_fit()` method, is that ok? 

This would be until I can get Disk Dataset working of course